src/hooks/fetchDoc.ts

```diff
@@ -22,11 +22,13 @@ function fetchDoc() {
         updateLoadingDoc(true)
         const path = window.location.pathname.toLowerCase()
         const usernamePath = path.split('/')[1].split('.')[0]
```

Comment:
what happens here if there is no "."? @DevPH-Wistkey

Reply:
we have that conditions on green line 25 @tralkan